File: solutions/JR_2023_F1_Epidemia.c
// NEPS 2316 - Epidemia (https://neps.academy/br/exercise/2316)
#include <stdio.h>

int main()
{
    int N, R, P;
    scanf("%d %d %d", &N, &R, &P);

    int dias = 0, infectadas = N;

    while (N < P)
    {
        infectadas = R*infectadas;
        N += infectadas;
        ++dias;
    }

    printf("%d\n", dias);

    return 0;
}

File: solutions/JR_2023_F1_Epidemia.py
# NEPS 2316 - Epidemia (https://neps.academy/br/exercise/2316)

N = int(input())
R = int(input())
P = int(input())

ans = 0
total = N

while total < P:
    N *= R
    total += N
    ans += 1

print(ans)
